packages/salesforcedx-sobjects-faux-generator/test/integration/fauxGenerate.test.ts

```diff
@@ -5,38 +5,31 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import { SFDX_PROJECT_FILE } from '@salesforce/salesforcedx-utils-vscode/out/src';
+import { AuthInfo } from '@salesforce/core';
```

Comment:
With these changes, we do not create a repo, create a scratch org, run describes against it, etc. For now, these tests assume we always get the same response shape from the org and we make sure that our code works as expected.

I'll be adding a system test that runs queries against an org to make sure we catch any api breaking changes as part of a follow up story.

Reply:
Good idea to add a system-test to catch any breaking changes